integration/test/sanity/sanity_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+// +build linux
+// +build integration
+
+package sanity
+
+import (
+	"github.com/aws/amazon-cloudwatch-agent/integration/test"
+	"testing"
+)
+
+func TestAgentStatus(t *testing.T) {
```

Comment:
Where it would trigger this file though? Just curiosity

Reply:
https://github.com/aws/amazon-cloudwatch-agent/blob/master/integration/terraform/ec2/linux/main.tf#L31